### ISSUE TITLE
Method to ignore official updates

### ISSUE THREAD

**Issue by Tarun80** [szepeviktor/fix-w3tc#568]
Hi,

This was present in previous versions but unfortunately did not make it into the more current builds. Is it possible for this option to be readded and/or favour the community version?

**Comment by Tarun80** [comment]
Anyone?

**Comment by szepeviktor** [comment]
Maybe https://github.com/szepeviktor/w3-total-cache-fixed/pull/127 ?

**Comment by Tarun80** [comment]
Yeah, that was definitely in 0.9.4, but sadly disappeared from 0.9.5+ going forward.
